packages/spcr-settings/settingsSection.tsx

```diff
@@ -77,7 +71,7 @@ class SettingsSection {
     if (!pluginSettingsContainer) {
       pluginSettingsContainer = document.createElement("div");
       pluginSettingsContainer.id = this.settingsId;
-      pluginSettingsContainer.className = styles.settingsContainer;
+      pluginSettingsContainer.className = "settingsContainer";
```

Comment:
You dropped `settings.module.css` so you removed `styles.settingsContainer`. But why replace it with `"settingsContainer"`? I don't think it does anything.